src/os_win/os_fs.c

```diff
@@ -17,7 +17,7 @@
 			ret = __wt_map_windows_error(windows_error);	\
 			if (windows_error == ERROR_ACCESS_DENIED) {	\
 				if (__retry == 0)			\
-					__wt_errx(session,		\
+					__wt_err(session, ret,		\
```

Comment:
This is the only one you've changed that I disagree with. This is an informational message that is printed on the first encounter of this and then we reset `ret` and retry a bunch of times. If the user eventually gets an access denied error, then an error will be returned to them after all the retries. This situation is often transient.

Reply:
ca5b267

That makes sense. Thanks for catching that.